app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java

```diff
@@ -175,14 +183,28 @@ private void buyTicket() {
     }
 
     private void addToCalendar() {
-        Intent intent = new Intent(Intent.ACTION_INSERT)
-                .setData(CalendarContract.Events.CONTENT_URI)
-                .putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, event.getDate().getTime())
-                .putExtra(CalendarContract.EXTRA_EVENT_END_TIME, event.getDate().getTime() + 7200000)
-                .putExtra(CalendarContract.Events.TITLE, event.getTitle())
-                .putExtra(CalendarContract.Events.DESCRIPTION, event.getDescription())
-                .putExtra(CalendarContract.Events.EVENT_LOCATION, event.getLocality())
-                .putExtra(CalendarContract.Events.AVAILABILITY, CalendarContract.Events.AVAILABILITY_FREE);//Indicates that this event is free time and will not conflict with other events.
+        // TODO: choose between TUM Calendar and external calendar app
+        boolean tumCalendar = true;
+        Intent intent;
+        if(tumCalendar){
+            intent = new Intent(context, CreateEventActivity.class);
+            intent.putExtra(Const.EVENT_EDIT, false);
```

Comment:
We should also consider the case where the user has already added the event to the calendar, then the edit property should be true right? Or alternatively, we could just take the user to the corresponding day in the calendar.

Reply:
I am not sure if this is feasible. How can we reliably check if the event has already been created? The user might have manually adjusted time and/or title.